src/passes/OptimizeInstructions.cpp

```diff
@@ -133,6 +133,69 @@ struct LocalScanner : PostWalker<LocalScanner> {
   }
 };
 
+// perform some final optimizations
```

Comment:
Please add
```cpp
namespace {
```
before, and
```cpp
} // anonymous namespace
```
after this class. Without them, if another file defines something with the same name, they will be linked together. (This is only a problem in the top level, that is, the global scope.)

Reply:
Done!